docs/extend/modular-resources/examples/odrive.md

```diff
@@ -0,0 +1,281 @@
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="odrive-canbus" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+//  < INSERT YOUR ODRIVE MODULE CONFIG >
+{
+  "components": [
+    {
+      "model": "viam:motor:odrive-canbus",
+      "namespace": "rdk",
+      "attributes": {
+        "canbus_node_id": <int>,
+      },
+      "depends_on": [],
+      "type": "motor",
+      "name": "<your-odrive-motor>"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "modules": [
+    {
+      "name": "odrive",
+      "executable_path": "/path/to/run.sh"
+    }
+  ],
+  "components": [
+    {
+      "model": "viam:motor:odrive-canbus",
+      "namespace": "rdk",
+      "attributes": {
+        "canbus_node_id": 0,
+        "odrive_config_file": "/path/to/first/config.json",
+        "serial_number": "NUM0001"
+      },
+      "depends_on": [],
+      "type": "motor",
+      "name": "odrive-motor"
+    },
+    {
+      "model": "viam:motor:odrive-canbus",
+      "namespace": "rdk",
+      "attributes": {
+        "canbus_node_id": 2,
+        "odrive_config_file": "/path/to/second/config.json",
+        "serial_number": "NUM0002"
+      },
+      "depends_on": [],
+      "type": "motor",
+      "name": "odrive-motor-2"
+    }
+  ]
+}
+```
+
+Edit and fill in the attributes as applicable to your model of ODrive.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for the motor resources available in the Viam ODrive module:
+
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `canbus_node_id` | int | Optional | Required for successful initialization of the `"odrive-canbus"` type. <br> Node ID of the CAN node you would like to use. You configured this when [setting up your ODrive](https://docs.odriverobotics.com/v/latest/can-guide.html#setting-up-the-odrive). <br> Example: `0` |
+| `odrive_config_file` | string | Optional | Filepath of a separate JSON file containing your ODrive's native configuration. </br> See the [Viam Github](https://github.com/viamrobotics/odrive/tree/main/sample-configs) for an example of this file. |
+| `serial_number` | string | Optional | The serial number of the ODrive. Note that this is not necessary if you only have only ODrive connected. See [Troubleshooting](#hanging) for help finding this value. |
+| `canbus_baud_rate` | string | Optional | [Baud rate](https://docs.odriverobotics.com/v/latest/can-guide.html#setting-up-the-odrive) of the ODrive CAN protocol. This attribute is only available for `"odrive-canbus"` connections. </br> Use [`odrivetool`](https://docs.odriverobotics.com/v/latest/odrivetool.html) to obtain this value with `<odrv>.can.config.baud_rate`. Format the string as a multiple of 1000 (k). <br> Example: `"250k"` |
+
+Save the config.
+Check the **Logs** tab of your robot in the Viam app to make sure your ODrive motor has connected and no errors are being raised.
```

Comment:
This is a good callout - can you add a link to the logs tab to the docs page where we describe the logs tab?

Reply:
done